pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -564,7 +564,7 @@ def channel_contributions_forward_pass(
             excluded=[1, 2],
             signature="(m, n) -> (m, n)",
         )
-        return target_transformed_vectorized(channel_contribution_forward_pass)
+        return target_transformed_vectorized(channel_contribution_forward_pass).eval()
```

Comment:
I think you need to put the eval inside, since `target_transformed_vectorized` is pure numpy code

```suggestion
        return target_transformed_vectorized(channel_contribution_forward_pass.eval())
```

Reply:
@juanitorduz can we use PyTensor functions for the `inverse_transform`s? That would simplify this PR quite a lot. We could use `pytensor.graph.vectorize` instead of `numpy.vectorize`